src/database/migrations/20230327061757-create-cart.js

```diff
@@ -8,10 +8,10 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER,
       },
-      user_id: {
+      userId: {
         type: Sequelize.STRING,
```

Comment:
should be an integer

Reply:
Have to create another migration to alter table carts with user_id of integer data type.